jetbrains-ultimate/tst-201+/software/aws/toolkits/jetbrains/datagrip/auth/SecretsManagerAuthTest.kt

```diff
@@ -63,45 +68,71 @@ class SecretsManagerAuthTest {
     fun `Intercept credentials succeeds`() {
         createSecretsManagerClient()
         val connection = sAuth.intercept(buildConnection(), false)?.toCompletableFuture()?.get()
-        assertThat(connection).isNotNull
-        assertThat(connection!!.connectionProperties).containsKey("user")
+        assertNotNull(connection)
```

Comment:
i thought we were killing kotlintest?

Reply:
`assertNotNull` uses contracts. We could write our own but not using assertj syntax (My PR to ban kotlin test explicitly allows only assertNotNull)